### ISSUE TITLE
Unexpected error occur after updating Powershell.

### ISSUE THREAD

**Issue by kashyap2003** [PowerShell/PSReadLine#2997]
### Prerequisites

- [X] Write a descriptive title.
- [X] Make sure you are able to repro it on the [latest released version](https://www.powershellgallery.com/packages/PSReadLine)
- [X] Search the existing issues, especially the pinned issues.

### Exception report

```console
System.TypeLoadException: Could not load type 'System.Management.Automation.Subsystem.PredictionResult' from assembly 'Microsoft.PowerShell.PSReadLine.Polyfiller, Version=1.0.0.0, Culture=neutral, PublicKeyToken=null'.
   at Microsoft.PowerShell.PSConsoleReadLine.PredictionViewBase.Reset()
   at Microsoft.PowerShell.PSConsoleReadLine.PredictionInlineView.Reset()
   at Microsoft.PowerShell.PSConsoleReadLine.Prediction.Reset()
   at Microsoft.PowerShell.PSConsoleReadLine.Initialize(Runspace runspace, EngineIntrinsics engineIntrinsics)
   at Microsoft.PowerShell.PSConsoleReadLine.ReadLine(Runspace runspace, EngineIntrinsics engineIntrinsics, CancellationToken cancellationToken)
```

### Screenshot

![Screenshot (5)](https://user-images.githubusercontent.com/77486001/141077141-b55173bf-2659-48af-acda-86a055dae5d4.png)

### Environment data

```console
PSReadLine: 2.2.0-beta1
PowerShell: 7.2.0
OS: Microsoft Windows 10.0.22000
BufferWidth: 120
BufferHeight: 30
```

### Steps to reproduce

Don't Know.

### Expected behavior

![Screenshot (6)](https://user-images.githubusercontent.com/77486001/141077734-59a1d18d-e5b6-4039-a3ff-140ad6dd2ab8.png)

### Actual behavior

![Screenshot (4)](https://user-images.githubusercontent.com/77486001/141077856-66c5850e-80e4-44da-86e6-ecca6c065afc.png)

**Comment by bulax** [comment]
Please update Psreadline to latest version 2.2.0-beta4

**Comment by cosmo0** [comment]
> Please update Psreadline to latest version 2.2.0-beta4

In case like me you're wondering how, follow the installation steps here: https://github.com/PowerShell/PSReadLine

**Comment by DominikIlski** [comment]
When i try to run
`Install-Module PSReadLine -RequiredVersion 2.2.0-beta4 -AllowPrerelease`
i get 
```
Install-Package: Administrator rights are required to install or update. Log on to the computer with an account that has Administrator
rights, and then try again, or install by adding "-Scope CurrentUser" to your command. You can also try running the
Windows PowerShell session with elevated rights (Run as Administrator).
```
even though i run powershell as an Admin.

**Comment by cosmo0** [comment]
Yeah I got the same, and I followed the directions in the error message: add `-Scope CurrentUser` to the `Install-Module` command. Then you have to restart Powershell.

**Comment by DominikIlski** [comment]
I have tried this flag before, but with no succes, but now it is working.
To solve this issue just run:
`Install-Module PSReadLine -RequiredVersion 2.2.0-beta4 -AllowPrerelease -Scope CurrentUser`
And restart Powershell

**Comment by daxian-dbw** [comment]
This issue was fixed in 2.2.0-beta3 version of PSReadLine. You can fix this by upgrading to the latest [2.2.0-beta4 version of PSReadLine](https://www.powershellgallery.com/packages/PSReadLine/2.2.0-beta4). Instructions for doing so:
1. stop all instances of `pwsh`.
2. from `cmd.exe` run: `pwsh -noprofile -command "Install-Module PSReadLine -AllowPrerelease -Force"`

--------

If you want to remove that beta version of PSReadLine and use the 2.1.0 version of PSReadLine that's shipped with PowerShell 7.2, you can:
1. run `pwsh -noprofile -noninteractive` to start `pwsh` without loading PSReadLine
2. run `Uninstall-Module -Name PSReadLine -RequiredVersion <2.2.0-beta1 or 2.2.0-beta2> -AllowPrerelease` to remove the module. Or, you can manually remove that module folder.
